File: src/00/sigma_core/src/ontology_gen/06/get_glyph_energy.rs
// SSoT: file:///Users/s0fractal/OMEGA/I/core/get_glyph_energy.md
#![allow(unused_imports)]
use super::super::L05::*;

pub fn get_glyph_energy(id: u8) -> u8 {
    GLYPH_ENERGY_LUT[(id & 63) as usize]
}

File: src/_/06/get_glyph_energy.ts
// SSoT: file:///Users/s0fractal/OMEGA/src/ontology/core/get_glyph_energy.md
import { GLYPH_ENERGY_LUT } from "@g05";

export function get_glyph_energy(id: number): number {
  return GLYPH_ENERGY_LUT[id & 63];
}
